examples/ex3p_o1.cpp

```diff
@@ -0,0 +1,337 @@
+//                       MFEM Example 3 - Parallel Version
```

Comment:
I get a segmentation fault when I run the following:

`./ex3p_o1 -m ../data/fichera-pyr.mesh`

Reply:
@brendankeith, sorry about that I had the wrong example on GitHub. The parallel version of example 3 uses HypreAMS which creates an arbitrary order H1 space internally for its discrete gradient operator. This is not yet available so it crashes while processing the pyramid shaped elements.

I've removed ex3p_o1.cpp and replaced it with ex3_o1.cpp which should work fine.